### DISCUSSION TITLE
Bizonylatkép vs. adattartalom

### DISCUSSION THREAD

**Discussion by laurelkft** [nav-gov-hu/eRECEIPT#7] [Kérdések és válaszok]
A nyugtának több partíciója is létezik, és ezek tárolódnak az enyugtatárban.
Fontos kérdés, hogy a másolatok nyomtatásánál, (kasszán, vagy bárhol máshol) a nyugtamásolat alatt a bizonylatképet értjük, vagy az eredeti adattartalommal kiállított másolati dokumentumot. 

Lehet-e egy nyugtamásolatnak több különböző bizonylatképe ha a kaszsán először 80mm-es papírra nyomtatjuk, vagy később (pl. mobilalkalmazásból) egy A4-es nyomtatóra.

**Comment by renced42** [comment]
Kedves @laurelkft   A bizonylat képnek egyformának kell lennie, függetlenül attól, hogy honnan nyomtatjuk ki. A nyugta másolatnak a beküldött elektronikus bizonylat adattartalmát kell vissza adnia. Ez alól kivételt képeznek az enumok amelyeket közérthetően kell feltüntetni a bizonylaton amennyiben az megjelenítendő.
